vault/example/Makefile

```diff
@@ -0,0 +1,22 @@
+
+VAULT_ADDR='http://127.0.0.1:8200'
```

Comment:
make this conditional (defaulted to) with `VAULT_ADDR?='http://127.0.0.1:8200'`

Reply:
done